src/test/java/org/fundacionjala/coding/omar/movies/CustomerTest.java

```diff
@@ -0,0 +1,135 @@
+package org.fundacionjala.coding.omar.movies;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Powered by Omar Limbert Huanca Sanchez - AT06.
+ * This class contains UnitTests for Movies App.
+ */
+public class CustomerTest {
+    private static final int NUMBER_7 = 7;
+    private static final int NUMBER_5 = 5;
+    private static final int NUMBER_10 = 10;
+    private static final int NUMBER_2 = 2;
+    private static final double NUMBER_6_5 = 6.5;
+    private static final int NUMBER_1 = 1;
+    private static final int NUMBER_30 = 30;
+    private static final double NUMBER_1_5 = 1.5;
+    private static final int CAPACITY = 100;
+    private Customer customer = new Customer("Pepito Perez");
+
+
+    /**
+     * This method is for test overViewNoRentedMovies().
+     */
+    @Test
+    public void testGetRentalOverviewNoRentedMovies() {
+        StringBuilder expectedOverviewBuilder = new StringBuilder(CAPACITY);
+
+        expectedOverviewBuilder.append("Customer: Pepito Perez"
+                + "\n====================================="
+                + "\nTotal 0.0\nYou earned 0 frequent renter point(s)");
+
+        assertEquals(expectedOverviewBuilder.toString(), customer.getRentalOverview());
+    }
+
+    /**
+     * This method is for test to add Children movie for 2 days.
+     */
+    @Test
+    public void testAddRentalChildrenChildrenForTwoDays() {
+        MovieChildren childrensMovie = new MovieChildren("The Minions");
+        Rental rental = new Rental(childrensMovie, NUMBER_2);
+
+        customer.addRental(rental);
+
+        assertRentalOverview(childrensMovie, NUMBER_1_5, NUMBER_1);
+    }
+
+    /**
+     * This method is for test to add New Movie for 10 days.
+     */
+    @Test
+    public void testAddRentalMovieNewForTenDays() {
+        MovieNew newReleaseMovie = new MovieNew("Black Panther");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Black Panther" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)